R/list_datasets.R

```diff
@@ -1,10 +1,11 @@
 #' Get a vector of timeseries variable names
 #' 
-#' get_ts_variables() returns a list of all possible timeseries variables, while
-#' get_ts_variables(site) returns the data available to a specific site.
+#' list_datasets() returns a list of all possible timeseries variables, while
```

Comment:
Ok, so we assume all "datasets" are timeseries? or will this also include things like categorical coverages (land-cover) or the like?